grabdish/inventory-springboot/pom.xml

```diff
@@ -1,44 +1,277 @@
+		</dependency>
+		<dependency>
+			<groupId>javax.transaction</groupId>
+			<artifactId>jta</artifactId>
+			<version>1.1</version>
+		</dependency>
+		<dependency>
+			<groupId>javax.transaction</groupId>
+			<artifactId>javax.transaction-api</artifactId>
+			<version>1.2</version>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.xml.bind</groupId>
+			<artifactId>jaxb-api</artifactId>
+			<version>2.4.0-b180830.0359</version>
+		</dependency>
+
+
+		<dependency>
+			<groupId>com.oracle.database.messaging</groupId>
+			<artifactId>aqapi</artifactId>
+			<version>19.3.0.0</version>
+		</dependency>
+		<!-- Java8 <dependency> <groupId>com.oracle</groupId> <artifactId>ojdbc8</artifactId> 
+			<version>18.4</version> </dependency> -->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ucp</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>rsi</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.security</groupId>
+			<artifactId>oraclepki</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.security</groupId>
+			<artifactId>osdt_core</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.security</groupId>
+			<artifactId>osdt_cert</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.ha</groupId>
+			<artifactId>simplefan</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.ha</groupId>
+			<artifactId>ons</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.xml</groupId>
+			<artifactId>xdb</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.xml</groupId>
+			<artifactId>xmlparserv2</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc.debug</groupId>
+			<artifactId>ojdbc11_g</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc.debug</groupId>
+			<artifactId>ojdbc11dms_g</artifactId>
+			<version>${version.lib.ojdbc}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.observability</groupId>
+			<artifactId>dms</artifactId>
```

Comment:
I think there are some redundancies re the oracle dependencies here but we can address this when we apply 21.3 versions soon